Wangscape/TilesetGenerator.cpp

```diff
@@ -102,4 +89,23 @@ void TilesetGenerator::generateClique(const Options::Clique& clique, sf::RenderT
             }
         }
     }
-}
\ No newline at end of file
+}
+
+std::string TilesetGenerator::getOutputImageFilename(const Options::Clique& clique) const
+{
+    std::stringstream ss;
+    for (auto terrain : clique)
+    {
+        ss << terrain << ".";
+    }
+    ss << options.fileType;
+    return ss.str();
+}
```

Comment:
Oh, now it can be tested! I didn't think about it, need to add some basic tests.